feyre_backend/sync_service/app.py

```diff
@@ -0,0 +1,23 @@
+import atexit
+
+from flask import Flask
+from sync_service.api.sync_user import sync_api, sync_users
+from apscheduler.schedulers.background import BackgroundScheduler
+
+sync_interval_seconds = 7200 # 2 Hours
```

Comment:
```suggestion
sync_interval_seconds = 3600 # 1 Hour
```

Shall this be reduced to one hour?